### PR TITLE
chore(readme): added new translation

### PR DESCRIPTION
**Pull request by 1998code**
Added new zh-hk translation
Fixed incorrect languages order in zh-cn and line breaking in zh-cn and ar-dz.

### CODE REVIEW

**Comment by 1998code** [comment]
Thanks :)

**Review by amyshwang — CHANGES_REQUESTED** [review]
Thank you for the translation. I have made some suggestions mainly based on usage in Taiwan - I hope it makes sense.

File: i18n/README.zh-hk.md [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+## 貢獻
+
+我們總是樂於接受對項目的貢獻，無論是源代碼、文檔、錯誤報告、功能請求還是反饋。如
+果要開始貢獻：
+
+- 請看一下 GitHub 上標有
+  "[Good first issue](https://github.com/questdb/questdb/issues?q=is%3Aissue+is%3Aopen+label%3A%22Good+first+issue%22)"
+  的問題。
+- 閱
+  讀[貢獻指南](https://github.com/questdb/questdb/blob/master/CONTRIBUTING.md)。
```
[the remark this answers is not in the document]
Comment (1998code) [reply i18n/README.zh-hk.md]: Fixed :)

File: i18n/README.zh-hk.md [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+
+- [AWS AMI](https://questdb.io/docs/guides/aws-official-ami)
+- [Google Cloud Platform](https://questdb.io/docs/guides/google-cloud-platform)
+- [Official Docker image](https://questdb.io/docs/get-started/docker)
+- [DigitalOcean droplets](https://questdb.io/docs/guides/digitalocean)
+- [Kubernetes Helm charts](https://questdb.io/docs/guides/kubernetes)
+
+## 貢獻
+
+我們總是樂於接受對項目的貢獻，無論是源代碼、文檔、錯誤報告、功能請求還是反饋。如
```
[the remark this answers is not in the document]
Comment (1998code) [reply i18n/README.zh-hk.md]: Changed :)

File: i18n/README.zh-hk.md [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+  能應用。
+- [產品路線圖](https://github.com/questdb/questdb/projects/3)列出了我們目前正在
+  進行的任務和功能。
+
+### ❓ 尋求支持
+
+- [Community Slack:](https://slack.questdb.io) 是一個進行技術討論和認識其他用戶
+  的好地方。👋
+- [GitHub issues:](https://github.com/questdb/questdb/issues) 報告 QuestDB 缺陷
+  或是反饋問題。
```
[the remark this answers is not in the document]
Comment (1998code) [reply i18n/README.zh-hk.md]: Changed :)

File: i18n/README.zh-hk.md [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+
+```bash
+brew install questdb
+brew services start questdb
+
+questdb start // 啓動 questdb
+questdb stop  // 停止 questdb
+```
+
+[QuestDB 下載頁面](https://questdb.io/get-questdb/) 提供二進制文件的直接下載，並提供其他安裝和部署方法的詳細信息。
```
[the remark this answers is not in the document]
Comment (1998code) [reply i18n/README.zh-hk.md]: Changed :)

File: i18n/README.zh-hk.md [hunk 5]
```diff
@@ -0,0 +1,296 @@
...
+  <a href="https://questdb.io/time-series-benchmark-suite/">
+    <img
+      alt="A chart comparing the maximum throughput of QuestDB, ClickHouse, TimescaleDB and InfluxDB."
+      src="https://raw.githubusercontent.com/questdb/questdb/master/.github/tsbs-results.png"
+    />
+  </a>
+</div>
+
+下表顯示了在 `c5.metal` 實例上使用 96 個線程中的 16 個線程運行 10 億條記錄的查詢
+執行時間。
```
[the remark this answers is not in the document]
Comment (1998code) [reply i18n/README.zh-hk.md]: Okay :)

**Comment by 1998code** [comment]
Thanks @amyshwang :)

**Pull request merged**